### DISCUSSION TITLE
update a single plugin version

### DISCUSSION THREAD

**Discussion by dezren39** [nix-community/nixvim#642] [Q&A] [answered]
sorry i'm not sure if this is obvious, but how do i, say, use the master branch of harpoon for a while?

https://github.com/ThePrimeagen/harpoon/pull/321

is it like installing a custom plugin or what do i do?

**Comment by GaetanLepage** [comment, marked as answer]
The current situation is a bit embarrassing. Unfortunately, the vim plugins are lacking on nixpkgs, mostly due to a failed refactor of the update script.
We are working with upstream to update the vim plugins ASAP.
In the meantime, if you use nixvim as a HM/NixOS module, I would advise you to roll back to a working revision of `nixpkgs`.
I am personally using `f99e5f03cc0aa231ab5950a15ed02afec45ed51a` and have a working harpoon.

Sorry for the inconvenience, but we cannot do much more here.

**Reply by dezren39** [reply]
thank you very much! i will delay updating!

**Reply by dezren39** [reply]
i see that harpoon is updated in master just not to unstable yet! i also see the general vim plugins pr update working its way through. :-)

**Reply by GaetanLepage** [reply]
The other vim plugins update just got merged 1 minute ago.
Things will get better as those updates land in nixos-unstable !
